pkg/migrations/mls/20240528181851_init-schema.up.sql

```diff
@@ -0,0 +1,71 @@
+SET statement_timeout = 0;
```

Comment:
I consolidated all our MLS schema into one big migration after the drops. Just easier to read this way